aiohttp/test_utils.py

```diff
@@ -632,7 +633,7 @@ def make_mocked_request(
     protocol.writer = writer
 
     if payload is sentinel:
-        payload = mock.Mock()
+        payload = EMPTY_PAYLOAD
```

Comment:
Feels to me like everything here should be a mock. Maybe instead we should autospec it? That should ensure it gets an appropriate mock with the correct methods.
```suggestion
        payload = create_autospec(StreamResponse, spec_set=True, instance=True)
```

Reply:
Same should really be done to all the rest of this code... Would make the code safer and more maintainable.

Reply:
Nevermind, seems to block the read indefinitely. This seems fine then.